ansible/library/ptf_portchannel.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+
+DOCUMENTATION = '''
+module:  ptf_portchannel
+version_added:  "1.0"
```

Comment:
This `version_added` field will be interpreted by ansible that the `ptf_portchannel` module is added in ansible version 1.0. Suggest to remove this field.

Reply:
Ok, remove it.